crates/nu-std/tests/run.nu

```diff
@@ -79,13 +79,16 @@ def main [
     let tests = (
         ls ($path | path join $module_search_pattern)
         | each {|row| {file: $row.name name: ($row.name | path parse | get stem)}}
-        | upsert test {|module|
-            nu -c $'use ($module.file) *; $nu.scope.commands | select name module_name | to nuon'
+        | upsert commands {|module|
+            nu -c $'use `($module.file)` *; $nu.scope.commands | select name module_name | to nuon'
```

Comment:
why backticks here? :yum:

Reply:
ooh yeah, you're right, i've already seen such things with windows actually :+1: